File: Ad-Hoc/2420/2420.c
#include <stdio.h>

int main(){
    int n, soma = 0, indice = 0, aux = 0;
    scanf("%d", &n);
    int lista[n];
    for (int i = 0; i < n; i++){
        scanf("%d", &lista[i]);
        soma += lista[i];
    }
    soma /= 2;
    for (int j = 0; j < n; j++){
        aux += lista[j];
        if (aux == soma){
            indice = j+1;
            break;
        }
    }
    printf("%d\n", indice);
    return 0;
}

File: Ad-Hoc/2420/2420.py
N = int(input())
if N>=1 and N<=10**5:
    lista = input().split()
    lista_int = []
    soma_lista = 0
    soma_divisao = 0
    indice = 0
    for x in lista:
        lista_int.append(int(x))
        soma_lista += int(x)
    divisao = soma_lista//2
    for y in range(len(lista_int)):
        soma_divisao += lista_int[y]
        if soma_divisao==divisao:
            indice = y
    print(indice+1)